docs/api/networkinterfaces/networkinterface.md

```diff
@@ -1,8 +1,8 @@
 ## Network interface overview
 
-A socket requires a NetworkInterface instance when opened to indicate which NetworkInterface the socket should be created on. The NetworkInterface provides a network stack that implements the underlying socket operations.
+A socket requires a NetworkInterface instance to indicate which NetworkInterface the socket should be created on. The NetworkInterface provides a network stack that implements the underlying socket operations.
```

Comment:
Query: Are we talking about the class every time we use NetworkInterface and the concept every time we use network interface? Please confirm.

Reply:
Yes.
When NetworkInterface is used, it means instance of the class. So maybe it should be `NetworkInterface` because it refers to API.

When "network interface" is instead used, we refer to actual physical interface.